src/models/ActivityModel.ts

```diff
@@ -0,0 +1,24 @@
+import Joi from 'joi';
+import { ActivityType } from '@dvsa/cvs-type-definitions/types/v1/enums/activityType.enum';
+import { WaitReason } from '@dvsa/cvs-type-definitions/types/v1/enums/waitReason.enum';
+import { TestStationTypes } from '@dvsa/cvs-type-definitions/types/v1/enums/testStationType.enum';
+
+export const ActivityModel = Joi.object().keys({
```

Comment:
Why has this been re-named to be activity model, it isn't a model it's a validation schema?

Reply:
Was discussed in the PR chat. differentiates it from the type `ActivitySchema`